contractgen/src/test/projects/demo/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+   Copyright 2014 Linagora, Université Joseph Fourier
```

Comment:
Can you explain the license here ?
Where is come from, we are not supposed to just copy some open source code directly even for testing.

Reply:
my fault, the demo is written by myself, but I forgot to update the license of template file.It is 
 already fixed now.